bodhi/server/consumers/masher.py

```diff
@@ -958,12 +958,24 @@ def compose_atomic_trees(self):
                 release['repos']['updates'] = mash_path
 
             # Compose the tree, and raise an exception upon failure
+            notifications.publish(topic="ostree.start",
+                                  msg=dict(tag=tag),
+                                  force=True)
             result = composer.compose(release)
             if result['result'] != 'success':
                 self.log.error(result)
+                notifications.publish(topic="ostree.start",
```

Comment:
I would prefer just keeping with the ==success !=success cases, as those are the only two that the atomic composer is aware of.